apps/hash-agents/pyproject.toml

```
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "hash-agents"
+version = "0.0.0"
+description = ""
+license = "GNU Affero General Public License v3.0"
```

not one of the recommended notations: https://python-poetry.org/docs/pyproject/#license

---

Probably does not matter as we're not going to publish this, but good to follow their recommendations 🙂 
```suggestion
license = "AGPL-3.0-only"
```